test/integration/integration_test.go

```diff
@@ -234,6 +315,8 @@ func selectFieldFromLastOutput(commandField string) string {
 		outputField = lastCommandOutput.StdErr
 	case "exitcode":
 		outputField = strconv.Itoa(lastCommandOutput.ExitCode)
+	case "proxy log":
```

Comment:
This is not the right place, as this function should only deal with the last command output, which is a typed struct which contains command specific output.

It is probably better to define a specific function to deal with this instead. This might mean that you need to define a step definition as:

`"^proxy log should contain", content` instead... which can then call it's own function.